src/assets/translations/en-US.json

```
@@ -682,6 +682,7 @@
       "Inactivate": "Inactivate",
       "Initiate Dividend": "Initiate Dividend",
       "Is Self Service": "Is Self Service",
+      "Link Client": "Link Client",
```

Biome (2.5.6)

[error] The key Link Client was already declared.

(lint/suspicious/noDuplicateObjectKeys)